jumpscale/sals/nginx/templates/location.conf

```diff
@@ -1,4 +1,4 @@
-location {{ location.path_url }} {{ "/" if not location.path_url.endswith("/") }} {
```

Comment:
make sure in threebot that we do `rstrip("/")` for `path_url` and add `/` here at the end